saleor/static/dashboard-next/productTypes/components/ProductTypeAttributes/ProductTypeAttributes.tsx

```diff
@@ -0,0 +1,182 @@
+        <>
+          <Card>
+            <CardTitle
+              title={
+                type === AttributeTypeEnum.PRODUCT
+                  ? i18n.t("Product Attributes")
+                  : i18n.t("Variant Attributes")
+              }
+              toolbar={
+                <Button
+                  color="secondary"
+                  variant="flat"
+                  onClick={toggleAttributeAddDialog}
+                >
+                  {i18n.t("Add attribute", { context: "button" })}
+                </Button>
+              }
+            />
+            <Table>
+              <TableHead>
+                <TableRow>
+                  <TableCell>{i18n.t("Attribute name")}</TableCell>
+                  <TableCell className={classes.textLeft}>
+                    {i18n.t("Values")}
+                  </TableCell>
+                  <TableCell />
+                </TableRow>
+              </TableHead>
+              <TableBody>
+                {renderCollection(
+                  attributes,
+                  attribute => (
+                    <Toggle key={maybe(() => attribute.id)}>
+                      {(
+                        openedAttributeEditDialog,
+                        { toggle: toggleAttributeEditDialog }
+                      ) => (
+                        <>
+                          <TableRow
+                            className={!!attribute ? classes.link : undefined}
+                            hover={!!attribute}
+                            onClick={toggleAttributeEditDialog}
+                            key={maybe(() => attribute.id)}
+                          >
+                            <TableCell>
+                              {maybe(() => attribute.name) ? (
+                                attribute.name
+                              ) : (
+                                <Skeleton />
+                              )}
+                            </TableCell>
+                            <TableCell className={classes.textLeft}>
+                              {maybe(() => attribute.values) !== undefined ? (
+                                attribute.values
+                                  .map(value => value.name)
+                                  .join(", ")
+                              ) : (
+                                <Skeleton />
+                              )}
+                            </TableCell>
+                            <TableCell className={classes.iconCell}>
+                              <IconButton
+                                onClick={event =>
+                                  onAttributeDelete(attribute.id, event)
+                                }
+                              >
+                                <DeleteIcon color="secondary" />
+                              </IconButton>
+                            </TableCell>
+                          </TableRow>
+                          <ProductTypeAttributeEditDialog
```

Comment:
Do we need a separate edit dialog for each table row? What if we used the URL and subroutes to toggle dialogs like this one?